staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -710,6 +710,10 @@ type KubeletConfiguration struct {
 	// Default: nil
 	// +optional
 	KubeReserved map[string]string `json:"kubeReserved,omitempty"`
+	// This ReservedSystemCPUs option specifies the cpu list reserved for the host level system threads and kubernetes related threads.
+	// This provide a "static" CPU list rather than the "dynamic" list by system-reserved and kube-reserved.
+	// This option overwrites CPUs provided by system-reserved and kube-reserved.
+	ReservedSystemCPUs string `json:"reservedSystemCPUs,omitempty"`
```

Comment:
The field requires validation.

Reply:
good reminder. will add it in two place: validation.go and server.go. In validation phase check it is a valid cpu list string, also make sure user does not intent to use  --kube-reserved-cgroup nor  --system-reserved-cgroup with the specific cpulist; in server.go make sure the cpulist is supported by the machine. If none of these condition meet, always take the old behavior

Reply:
updated